docs/whatsnew/1.1.rst

```diff
@@ -0,0 +1,182 @@
+        HeliocentricMeanEcliptic -> ICRS
+        HeliocentricMeanEcliptic -> HeliocentricEarthEcliptic
+        HeliocentricMeanEcliptic -> GeocentricEarthEquatorial
+        HeliographicStonyhurst -> HeliographicCarrington
+        HeliographicStonyhurst -> Heliocentric
+        HeliographicStonyhurst -> HCRS
+        HeliographicStonyhurst -> HeliocentricInertial
+        HeliographicCarrington -> HeliographicStonyhurst
+        Heliocentric -> Helioprojective
+        Heliocentric -> HeliographicStonyhurst
+        Helioprojective -> Heliocentric
+        HeliocentricEarthEcliptic -> HeliocentricMeanEcliptic
+        HeliocentricEarthEcliptic -> GeocentricSolarEcliptic
+        GeocentricSolarEcliptic -> HeliocentricEarthEcliptic
+        HeliocentricInertial -> HeliographicStonyhurst
+        GeocentricEarthEquatorial -> HeliocentricMeanEcliptic
+        subgraph cluster_astropy {
+                color=blue
+                fontcolor=blue
+                penwidth=2
+                label=<<b>Frames implemented in Astropy</b>>
+                ICRS
+                HCRS
+                HeliocentricMeanEcliptic
+                astropy [label="Other Astropy frames" shape=box3d style=filled]
+                geocentric [label="Earth-centered frames (including GEO)" shape=box3d style=filled]
+                astropy -> ICRS
+                geocentric -> ICRS
+                ICRS -> astropy
+                ICRS -> geocentric
+        }
+        subgraph cluster_sunpy {
+                color=crimson
+                fontcolor=crimson
+                penwidth=2
+                label=<<b>Frames implemented in SunPy</b>>
+                Helioprojective
+                Heliocentric
+                HeliographicStonyhurst
+                HeliographicCarrington
+                subgraph cluster_sunpy11 {
+                        color=chocolate
+                        fontcolor=chocolate
+                        label=<<b>Requires SunPy 1.1</b>>
+                        HeliocentricInertial
+                        HeliocentricEarthEcliptic
+                        GeocentricSolarEcliptic
+                        GeocentricEarthEquatorial
+                }
+        }
+        newrank=true
+   }
+
+.. _whatsnew-1.1-dl_manager:
+
+Manager for Versioned Data Files
+================================
+
+SunPy 1.1 provides a data manager for versioning and caching remote files. Given the URL of a remote file and a SHA256 hash, the `~sunpy.data.manager.require` decorator can be added to functions that require specific data files from remote sources (e.g. computing instrument response functions). If the specified hash does not match that of the remote version, an exception is raised to make the user aware of any changes on the remote server or tampering with local files. Additionally, `~sunpy.data.cache` can be used to avoid re-downloading files that already exist on a user's local machine, thus saving disk space and internet bandwidth.
```

Comment:
Split this over a few lines? Also a link to the actual docs for doing all this would be nice.

Reply:
The docs for data manager are a little wonky because of the fact that `sunpy.data.manager` is an instance (like Fido) need to look into that.